seed/src/lib.rs

```diff
@@ -117,10 +117,10 @@ impl Default for NodeConfig {
 
 /// Seed node instance.
 pub struct Node {
-    /// Receiver end of user requests.
-    requests: chan::UnboundedReceiver<Request>,
     /// Sender end of user requests.
     handle: chan::UnboundedSender<Request>,
+    /// Receiver end of user requests.
+    requests: chan::UnboundedReceiver<Request>,
```

Comment:
A classic xla change :smile:

Reply:
Sich seiner Linie treu bleiben.

Also for the request, something had to be chanted, clippy otherwise complains about inconsistent struct init. :shrug: